sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -5683,6 +5683,12 @@ pi_result cuda_piextUSMGetMemAllocInfo(pi_context context, const void *ptr,
   return result;
 }
 
+pi_result piextUSMImport(void *ptr, size_t size, pi_context context) {
```

Comment:
we have something similar in CU, cudaHostRegister, cudaHostUnregister . should we expand the definition of the extension to cover the functionality offered by all backends?

Reply:
It is not quite equivalent to the cuda HostRegister APIs. Those make the address range accessible from host and device and in addition pin the host memory to physical memory. We should add "pinning" as a separate extension when it is supported by L0.

Reply:
thanks @rdeodhar . Now that we are moving to the UR, all PI interfaces should be designed taking into account all interfaces. Or maybe, something like UR extensions would help here. I will bring up this in the UR WGs.

Reply:
> Moreover, the L0 APIs are driver-experimental extensions, and when merged to the spec, they would likely be much different. i dont think that we should define a SYCL extension until the L0 interfaces are at least defined in the L0 spec.

Note that this is also an experimental SYCL extension.  We can replace it with a more general API later if the L0 API changes.

Reply:
@gmlueck : Even if this is an experimental extension, we need UR interfaces for it.